src/Orchard.Cms.Web/Modules/Orchard.OpenId/Orchard.OpenId.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DebugType>portable</DebugType>
+    <AssemblyName>Orchard.OpenId</AssemblyName>
+    <PackageId>Orchard.OpenId</PackageId>
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
```

Comment:
Yup, it works fine on my machine. I took a look at the CI build logs and the OpenID module compiles fine on AppVeyor. That said, if you really need this line to make it work, feel free to add it... it's harmless.